lti_consumer/templates/html/student.html

```diff
@@ -36,7 +37,9 @@ <h2 class="problem-header">
             aria-label="This section of the courseware is delivered by a 3rd party. As such, we cannot make any guarantees that this external resource supports the same level of accessibility."
         >
             % if launch_pane_html:
-                ${launch_pane_html}
+                ## it is intended that authors can use HTML in the launch_pane_html parameter
+                ## so we allow this to go through unescaped
+                ${launch_pane_html | n}
```

Comment:
Please update to:
```
${launch_pane_html | n, decode.utf8}
```
See http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#mako-filter-ordering-and-the-n-filter